eb_ssm/eb_ssm.py

```diff
@@ -30,6 +30,11 @@ def __init__(self):
             "Please specify a specific region in the command or eb configuration.",
         )
 
+        self.command = args.command or "bash -l"
```

Comment:
At this point, `"bash -l"` should get moved out to a constant.  (I'd put `BASH_COMMAND = "bash -l"` right below the imports)